puppet/modules/profiles/manifests/web.pp

```diff
@@ -60,4 +60,7 @@
     rsync_max_connections => $rsync_max_connections,
   }
   contain web::vhost::yum
+
+  class { 'web::vhost::stagingyum': }
+  contain web::vhost::yum
```

Comment:
Technically this is redundant. If you don't need parameters, go with the simple variant.
```suggestion
  contain web::vhost::yum
```